NitroxServer/Communication/Packets/Processors/EntityReparentedProcessor.cs

```diff
@@ -0,0 +1,57 @@
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic;
+using NitroxServer.GameLogic.Entities;
+
+namespace NitroxServer.Communication.Packets.Processors
```

Comment:
Filescoped namespace

Reply:
Probably this could help: https://www.meziantou.net/updating-your-project-to-use-file-scoped-namespaces.htm